airflow/compat/functools.pyi

```diff
@@ -19,6 +19,8 @@
 # This stub exists to work around false linter errors due to python/mypy#10408.
 # TODO: Remove this file after the upstream fix is available in our toolchain.
 
+from __future__ import annotations
```

Comment:
These should not be needed since `.pyi` file contents are always lazily evaluated.

Reply:
Isort is adding this automatically, and i don't think isort had a way to not do automatic imports on .pyi, so will probably just accept this

Reply:
Ah. Cool Wasn't sure about it :)

Reply:
Would setting `skip = .pyi` work?